src/components/sections/accordion.json

```diff
@@ -0,0 +1,30 @@
+{
+  "collectionName": "components_sections_accordions",
```

Comment:
Question:

Is accordion added to the pageSections dynamic zone?

Reply:
Currently yes